pipelines/op-release-notes.adoc

```diff
@@ -28,8 +28,10 @@ For questions and feedback, you can send an email to the product team at pipelin
 
 // Modules included, most to least recent
 include::modules/op-release-notes-1-0.adoc[leveloffset=+1]
+
```

Comment:
I was getting build errors because immediately after this someone opened a multi-line comment and didn't close it with four slashes.    I thought separating that chuck of text from the includes comment might help avoid that error getting re-introduced.

Reply:
I see the `unterminated comment block` in my branches and master, but I don't see it breaking the github build. I see that it causes a warning in asciibinder for me rather than an error. My preference is for the PR to only contain relevant changes to our files, unless they're blockers. 

I'll approve the review, but I thought it was important to check.